adaptive_scheduler/monitoring/opensearch_seeing.py

```diff
@@ -0,0 +1,90 @@
+from opensearchpy import OpenSearch
+from dateutil.parser import parse
+from datetime import datetime, timedelta
+
+
+class ConnectionError(Exception):
+    pass
+
+
+def get_seeing(resources, opensearch_url, os_index, os_excluded_observatories, lookback_minutes):
+    ''' Get seeing value from an opensearch index structured like the banzai qc index. Any opensearch index which has keys on telescope
+        , site, observatory, and seeing value should work though.
+    '''
+    os = OpenSearch(opensearch_url, http_compress=True)
```

Comment:
os is a built in python module, so probably best to avoid naming variables after that